base_rest_datamodel/restapi.py

```diff
@@ -1,6 +1,6 @@
 # Copyright 2020 ACSONE SA/NV
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
-
+# flake8: noqa
```

Comment:
@yankinmax Why this 'flake8: noqa` We must avoid global flake8 ignore

Reply:
I understood. Actually I forgot, that I left it here. Thanks for pointing.